LoopKit/Locked.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Locked.swift
+//  LoopKit
+//
+//  Copyright © 2018 LoopKit Authors. All rights reserved.
+//
+
+import os.lock
+
+
+public class Locked<T> {
+    private var _lock: UnsafeMutablePointer<os_unfair_lock>
```

Comment:
Might be simpler to implement Locked in terms of UnfairLock to deduplicate the initialization logic here.